### ISSUE TITLE
CSS parser error

### ISSUE THREAD

**Issue by raisen** [Sysmagine/SemanticDiff#13]
**Describe the Bug**
I'm getting a parse error diff'ng 2 CSS files.

**To Reproduce**
Diff 2 CSS files that contain keyframes with multiple selectors, such as:

```
@keyframes targettext2 {
  0%, 50% {
    color: #989898;
  }
  100%, 25% {
    color: #000;
  }
}

```

**Expected Behavior**
It should parse those CSS files.

**Actual Behavior**
I get the error and cannot continue diff'ing

**Screenshots**
![Screenshot (6)](https://user-images.githubusercontent.com/242645/229580232-94463d3d-ef5c-4b61-9ca9-badc39048642.png)

**SemanticDiff Version**
0.8.2

**VS Code Information**
```
Version: 1.76.2 (user setup)
Commit: ee2b180d582a7f601fa6ecfdad8d9fd269ab1884
Date: 2023-03-14T17:55:54.936Z
Electron: 19.1.11
Chromium: 102.0.5005.196
Node.js: 16.14.2
V8: 10.2.154.26-electron.0
OS: Windows_NT x64 10.0.22621
Sandboxed: No
```

**Comment by mmueller2012** [comment]
Thanks for reporting the bug! As you found out yourself, the CSS grammar did not allow multiple keyframe selectors per keyframe block. I have already changed the grammar and the fix will be part of the next release :smiley:

**Comment by mmueller2012** [comment]
We have finally released SemanticDiff 0.8.3 which should fix your bug. Sorry for the long delay, we made some larger changes to the core components and things took longer than anticipated. In the future we plan to create bug fix only releases to avoid such long waiting times. Anyways, can you confirm that it works for you?

**Comment by mmueller2012** [comment]
Closing this bug since it has been fixed for a while. @raisen please reopen if you run into any issues.
